src/modules/icmaa-competitions/pages/Competition.vue

```diff
@@ -39,6 +39,14 @@
           {{ $t('Done') }}
         </div>
         {{ $t('Thank you. We successfully received your data and will inform you about further steps.') }}
+        <div v-if="successHeadline" class="t-flex t-items-center t-text-1xl t-text-primary t-mb-2 t-mt-8">
+          {{ successHeadline }}
+        </div>
+        <div class="t-flex t-w-full">
+          <div class="lg:t-w-1/2">
+            <picture-component v-if="successImage" :src="successImage" :width="624" :height="312" :placeholder="true" :sizes="sizes" ratio="1:1" class="t-flex-1 t-self-start" />
+          </div>
+        </div>
```

Comment:
```suggestion
        <div class="t-flex t-w-full" v-if="successImage">
          <div class="lg:t-w-1/2">
            <picture-component :src="successImage" :width="624" :height="312" :placeholder="true" :sizes="sizes" ratio="1:1" class="t-flex-1 t-self-start" />
          </div>
        </div>
```
Sonst wird der Block darum auch angezeigt wenn kein Bild existiert.

Reply:
Oh ja! Danke